app/product_notices.go

```diff
@@ -66,7 +66,7 @@ func noticeMatchesConditions(config *model.Config, preferences store.PreferenceS
 
 	// check if notice date range matches current
 	if cnd.DisplayDate != nil {
-		now := time.Now().UTC()
+		now := time.Now().UTC().Truncate(time.Hour * 24)
```

Comment:
Why do we truncate to hour now?

Reply:
@ashishbhate that's to avoid problems like this:
1. PM set condition to "= 2020-03-04T00:00"
2. The user requests notices at 2020-03-04 15:00
3. This condition fails

So we are now truncating to the day to reduce time resolution